Formula/bzt.rb

```diff
@@ -21,9 +21,9 @@ class Bzt < Formula
   depends_on "cmake" => :build
   depends_on "ninja" => :build
   depends_on "rust" => :build
-  depends_on "cython"
+  depends_on "libcython"
```

Comment:
We should combine these formulae again, I think.

Reply:
Currently, PythonVersions audit prevents having a runtime Python in formula with other `depends_on "python@3.x" => ...` dependencies. Probably won't be able to combine these until audit is changed.